test-workspace/other-files/config.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable */
+import {
+  CONSTANTS,
+  IntermediateDendronConfig,
+  DendronSiteConfig,
+  Time,
+} from "@dendronhq/common-all";
+import { readYAML, writeYAML } from "@dendronhq/common-server";
+import fs from "fs-extra";
+import _ from "lodash";
+import path from "path";
+
+export class DConfig {
```

Comment:
can we add this to [[Test Workspace|dendron://dendron.docs/dev.ref.test-workspace]] as something that people can use to test non-note files?

Reply:
Good idea, added.